apps/admin/app/students/page.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import { Flex } from "@styled-system/jsx";
+import { Text } from "@wow-class/ui";
+import { studyApi } from "apis/study/studyApi";
+import useFetchStudents from "hooks/fetch/useFetchStudents";
+import { useAtom } from "jotai";
+import { useEffect, useState } from "react";
+import type { StudyListApiResponseDto } from "types/dtos/studyList";
+import isAdmin from "utils/isAdmin";
+
+import StudentList from "./_components/StudentList";
+import StudentsHeader from "./_components/StudentsHeader";
+import { studyAtom } from "./_contexts/StudyProvider";
+
+const StudentsPage = () => {
+  const [studyList, setStudyList] = useState<StudyListApiResponseDto[]>();
+  const [study, setStudy] = useAtom(studyAtom);
```

Comment:
p4;
특정 스터디가 선택되었다는 부분이 네이밍에서 명확하게 드러나면 좋을 거 같아요! 
```suggestion
  const [selectedStudy, setSelectedStudy] = useAtom(studyAtom);
```